test/js/third_party/postgres/postgres.test.ts

```diff
@@ -1,11 +1,12 @@
 import { test, expect, describe } from "bun:test";
+import { isBunCI } from "harness";
 import { Pool, Client } from "pg";
 import { parse } from "pg-connection-string";
 import postgres from "postgres";
 
 const CONNECTION_STRING = process.env.TLS_POSTGRES_DATABASE_URL;
 
-const it = CONNECTION_STRING ? test : test.skip;
+const it = CONNECTION_STRING ? test : test.skipIf(!isBunCI);
```

Comment:
i agree that ci should run this, but from outside contributors this test will fail as the environment doesnt load. has this been fixed?